CHANGELOG.md

```diff
@@ -3,6 +3,37 @@
 All notable changes to this project will be documented in this file. See
 [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.29.0](https://github.com/tinkoff/taiga-ui/compare/v2.28.0...v2.29.0) (2021-12-28)
+
+### Features
+
+- **addon-doc:** `LegendItem` add checkbox support ([#1145](https://github.com/tinkoff/taiga-ui/issues/1145))
+  ([140391b](https://github.com/tinkoff/taiga-ui/commit/140391bf3f3c517f461dff4276d3795bd9b84ef4))
+- **addon-doc:** `PieChart`/`RingChart` add animations ([#1133](https://github.com/tinkoff/taiga-ui/issues/1133))
+  ([1803970](https://github.com/tinkoff/taiga-ui/commit/1803970cd0cc6b8e3da86816d64cf82afbe0ed35))
+- **cdk:** use ReplaySubject for TuiDestroyService
+  ([1c00e78](https://github.com/tinkoff/taiga-ui/commit/1c00e78468d3518d83bfa464846ddd3226c1e4f9))
+- **kit:** `InputDate` add full support of `TUI_DATE_FORMAT` ([#1146](https://github.com/tinkoff/taiga-ui/issues/1146))
+  ([52d6472](https://github.com/tinkoff/taiga-ui/commit/52d647257b71409a407e8c00cec85cfc93c64970))
+- **kit:** `InputDateRange` add full support of `TUI_DATE_FORMAT`
+  ([#1155](https://github.com/tinkoff/taiga-ui/issues/1155))
+  ([f6c8d08](https://github.com/tinkoff/taiga-ui/commit/f6c8d080139bba3d55d8608311b9aca594aca4de))
+- **kit:** `InputDateTime` add full support of `TUI_DATE_FORMAT`
```

Comment:
why does it duplicate two points before? 🤔

Reply:
no duplicates, these are different components